torch/serialization.py

```diff
@@ -338,7 +338,16 @@ def save(obj, f, pickle_module=pickle, pickle_protocol=DEFAULT_PROTOCOL, _use_ne
            containing a file name
         pickle_module: module used for pickling metadata and objects
         pickle_protocol: can be specified to override the default protocol
-
+        _use_new_zipfile_serialization: saves the object as a zip file rather than a binary file
+    
+    Note:
+        `.pt` extension is used for most of the cases.
+    
+    Note:
+        The file extension does not matter while saving the object. The object can be saved using
+        .pt, .zip or any other extension. (You can literally save the object as .py file and 
+        it will just work as intended). 
```

Comment:
Let's combine these two as 
```
Note:
    A common PyTorch convention is to save tensors using either a .pt or .pth file extension.
```

I would leave out the "The object can be saved using
        .pt, .zip or any other extension. (You can literally save the object as .py file and 
        it will just work as intended). " part, we don't want users to save their files as .zip or .py and then attempt to run `winzip` or `python` on them.

Reply:
@zou3519 Recommending `*.pth` is not very nice, because `*.pth`, because "pth" is a special file extension for Python and can lead no nasty package importing errors: #14864 ...

Reply:
Sounds good, let's avoid recommending `*.pth` then. We should change our recommendation in the tutorials then: https://pytorch.org/tutorials/beginner/saving_loading_models.html, but that can be done separately.

Reply:
Done